vita3k/gui/src/manual.cpp

```diff
@@ -168,7 +168,7 @@ void draw_manual(GuiState &gui, EmuEnvState &emuenv) {
     ImGui::SetWindowFontScale(RES_SCALE.x);
 
     // Hide button when right click is pressed on mouse
-    if (!ImGui::IsAnyItemHovered() && ImGui::IsMouseClicked(0))
+    if (!ImGui::IsAnyItemHovered() && ImGui::IsMouseClicked(1))
```

Comment:
No touch it please 🙏

Reply:
No, I definitely prefer this

Reply:
I think this is a bit much more of a personal preference because of visual novels hiding UI when right clicking. It doesnt really matter that much and matters even less cause no one looks at the game manuals :sob:, why not both?

Reply:
like i explain revert this change please, right click corresponding long touch on psvita, and is so also same on android, left = touch, right = long touch 

So real way here is left touch, like it works on real psvita

Reply:
Leave the left click to hide the manual, I'm not used to it like for no reason

Leave the left click to hide the manual, I'm not used to it like for no reason
psvita works like this, the left click corresponds to a simple press of the touchscreen, the right click corresponds to a long press on psvita

and here is so same on android

Reply:
> I think this is a bit much more of a personal preference because of visual novels hiding UI when right clicking. It doesnt really matter that much and matters even less cause no one looks at the game manuals 😭, why not both mouse buttons?

Seemingly this is the only compromise, so I will do this.
Nothing will change for zang.

Reply:
you don't understand pk I did it like that, forget the right click, I don't even know why you want to put it there, when it's absolutely not practical.

and that on the vita, it is indeed a left click
Reminder, right click = Long press on android and psvita

The official manual app itself if it works, use the left click....